pkg/agent/openflow/pipeline.go

```diff
@@ -2465,7 +2466,7 @@ func (f *featureEgress) externalFlows() []binding.Flow {
 				Done(),
 			// This generates the flow to match the packets sourced from tunnel and destined for external network, then
 			// forward them to EgressMarkTable.
-			L3ForwardingTable.ofTable.BuildFlow(priorityLow).
+			L3ForwardingDefaultTable.ofTable.BuildFlow(priorityHigh).
```

Comment:
One question: traffic for AntreaIPAM to external network is never processed by Egress feature? @gran-vmv

Reply:
I think Egress + AntreaIPAM is not supported since AntreaIPAM requires noSNAT and noEncap.